entity-framework/core/querying/tags.md

```diff
@@ -68,6 +68,38 @@ FROM [People] AS [p]
 ORDER BY [p].[Location].STDistance(@__myLocation_0) DESC
 ```
 
+## Tagging with file name and line number
+
+Starting with EF Core 6.0, queries can be automatically tagged with the file name and line number where the LINQ query is defined in source code. This is done using the `TagWithCallSite()` method:
+
+[!code-csharp[Main](../../../samples/core/Querying/Tags/Program.cs#TagWithCallSite)]
```

Comment:
Inline the code sample, and do not make any changes under the samples directory.